application/Views/blocks/a.php

```diff
@@ -0,0 +1,14 @@
+<div class="block a">
```

Comment:
Please remove theses 'blocks' views. I realize they were used while you were testing, but they're not something we want to ship with the framework.